kqueen_ui/blueprints/ui/views.py

```diff
@@ -594,8 +594,7 @@ def handle(self):
             # Append provisioner ID to parameter name to make it unique
             parameters = {
                 k + '__' + provisioner['id']: v
-                for [k, v]
-                in _parameters.items()
+                for [k, v] in _parameters.items()
```

Comment:
I think the way it is now is the right way to break python generator expression into multiple lines

Reply:
Not sure if there is a unique right way for this. For me generators are more readable when they are one-line, and afaik that's on of their purposes (ideally). Also you don't break normal loops this way, so it's more usual when `for-in` are together.